DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/use_library/part1_native/src/use_native.cpp

```diff
@@ -37,33 +64,16 @@ int main() {
     std::cout << "Running on device: "
               << device.get_info<sycl::info::device::name>().c_str()
               << std::endl;
+    {
+      //write data to host-to-device hostpipe
```

Comment:
```suggestion
      // write data to host-to-device hostpipe
```
fix comment formatting (I won't fix every one as that will make way too many messages :))